cypress/integration/dictionary/AddBulkConceptsToDictionaryPage/addBulkConceptsToDictionary.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Add bulk concepts to an existing dictionary
+  Background:
+    Given the user is logged in
+
+  @dictionary
+  Scenario: The user should be able to add bulk concepts from their preferred source
+    Given the user is on the view dictionary concepts page
+     When the user clicks the "Add concepts" button
+      And the user selects "Import existing concept"
+      And the user selects "Add bulk concepts"
+     Then the user should be on the "Add concepts in bulk from CIEL" page
+
+  @dictionary
+  Scenario: The user should be able to add a single bulk concept from their preferred source
+    Given the user is on the "Add concepts in bulk from CIEL" page
+     When the user enters concept Id "1000"
+      And the user clicks the "Add concepts" button
+     Then the user is send to "Progress notifications" page
+      And the concept Id "1000" should be added to the dictionary
+      
+  @dictionary
+  Scenario: The user should be able to add multiple bulk concepts from their preferred source
+    Given the user is on the "Add concepts in bulk from CIEL" page
+     When the user enters concept Id "1000"
+      And the user enters concept Id "1001"
+      And the user enters concept Id "1002"
+      And the user clicks the "Add concepts" button
+      And the user is sent to the "Progress notifications" page
+     Then the user clicks on "view summary" button
+       And concept Id "1000" should be skippped
```

Comment:
Why should "1000" be skipped? Remember that each scenario (should) run in isolation from other scenarios, so if we're expecting that concept 1000 is already in the dictionary, then we should have an explicit "Given" step to address this. 

```
Given CIEL concept "1000" is already in the dictionary
  And the user is on the "Add concepts in bulk from CIEL" page